Debug `print` function should be `print!`

### What browsers are you seeing the problem on?

Other (please write below)
Brave

### Bug report for the website

I think all examples that use `print` function at https://docs.substrate.io/v3/runtime/debugging/ should be using `print!` (with a bang character) instead

### Steps to reproduce the problem

In the Substrate Debugging deocumentation here https://docs.substrate.io/v3/runtime/debugging/

It says that we can use Substrate's `print` function with
```
print("My Test Message");
```

But if I do that I get error
```
error[E0423]: expected function, found macro `print`
  --> node/src/chain_spec.rs:51:5
   |
51 |     print("My Test Message");
   |     ^^^^^ not a function
```

Whereas if I use a bang character by changing it to `print!("My Test Message");` then it works

--- a/v3/docs/03-runtime/j-debugging/index.mdx
+++ b/v3/docs/03-runtime/j-debugging/index.mdx
@@ -77,7 +77,7 @@ pub fn cause_error(origin) -> dispatch::DispatchResult {
 	// Check it was signed and get the signer. See also: ensure_root and ensure_none
 	let _who = ensure_signed(origin)?;
 
-	print("My Test Message");
+	print!("My Test Message");
 
 	match Something::get() {
 		None => {
@@ -134,14 +134,14 @@ use sp_runtime::print;
 
 // --snip--
 pub fn do_something(origin) -> DispatchResult {
-	print("Execute do_something");
+	print!("Execute do_something");
 
 	let who = ensure_signed(origin)?;
 	let my_val: u32 = 777;
 
 	Something::put(my_val);
 
-	print("After storing my_val");
+	print!("After storing my_val");
 
 	Self::deposit_event(RawEvent::SomethingStored(my_val, who));
 	Ok(())

